example.py

```diff
@@ -16,10 +17,16 @@
 # for example, this would be a good format - this needs to be generated at the stream edge.
 stream_id = datetime.datetime.today().strftime('%Y_%m_%d__%H_%M_%S') + "_exp1"
 
+# Optionally, specify REST server with interesting model:
+interestingnessModel = RestInterestingnessModel('http://localhost:5000/model/api/v0.1/evaluate')
+
 client = HasteStorageClient(stream_id,
                             'localhost',  # IP address of database server.
                             27017,
-                            auth)
+                            auth,
+                            interestingness_model=interestingnessModel,
+                            storage_policy=[(lambda x: x > 0.5, 'swift')],
```

Comment:
What's fiddly here is dealing with < vs. =< etc. Using a lambda means you can just type python code.

Reply:
Scrub this, will just have `(from, to, category)` - closed intervals are good enough